docs/source/API/core/parallel-dispatch/parallel_for.rst

```diff
@@ -81,6 +84,18 @@ More Detailed Examples are provided in the ExecutionPolicy documentation.
         Kokkos::finalize();
     }
 
+.. raw:: html
+
+   <a href="https://godbolt.org/z/q9h339vob" target="_blank">Edit on Compiler Explorer</a>
+
+.. note::
+   Code snippet taken directly from Kokkos Core (``hello_world_lambda.cpp``) + a link containing full source code of the example.
+
+.. ceinclude::  ../../../../../kokkos/example/tutorial/01_hello_world_lambda/hello_world_lambda.cpp
+  :language: cpp
+  :start-after: Kokkos::initialize
+  :end-before: Kokkos::finalize
```

Comment:
`helo_world_lambda.cpp` + an auto-generated link. See `ceincude.py` for details.

Reply:
Given the number of comments in this file, I would rather not include that one specifically. Other than that, it's a good idea to use examples we run through our CI. I could see that we just reduce the verbosity in these files and refer to `kokkos-tutorials` for more extensive tutorials.